### PR TITLE
doc: fix snapshot link

### PR DESCRIPTION
**Pull request by tbruyelle**

### CODE REVIEW

**Comment by albttx** [comment]
image

Thanks @tbruyelle for the fix, could you update the bucketname, i believe the mistake was more in the existance of atomEone :)

**Review by moul — APPROVED** [review]

**Comment by tbruyelle** [comment]
@albttx I added a second commit that fixes the bucket name already.

File: snapshots/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This is because after the tally is completed, the votes a removed from state.
 
 ### Download
 
-An export is stored in S3 accessible here: https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json
+An export is stored in S3 accessible here: https://atomeone.fra1.digitaloceanspaces.com/cosmoshub-4/cosmoshub-4-export-18010657.json
```
Comment (giunatale) [inline snapshots/README.md:13]: @tbruyelle @albttx can we fix this so the bucket has `atomone` in the name?

**Review by giunatale — CHANGES_REQUESTED** [review]

File: snapshots/README.md (hunk 1 above)
Comment (albttx) [reply snapshots/README.md:13]: Yes, you can revert, i've created the bucket `atomone`

Comment (tbruyelle) [reply snapshots/README.md]: commit reverted, thanks for the fix!

**Review by giunatale — APPROVED** [review]

**Pull request merged**